cpp-terminal/base.hpp

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include <string>
+
+namespace Term {
+    enum class style {
```

Comment:
This should be indented as:
```suggestion

enum class style {
```
Also let's add an empty line.